merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -338,7 +338,7 @@ private static void postActivities(
       final Timestamp simulationStart
   ) throws SQLException {
     try (
-        final var postActivitiesAction = new PostSpansAction(connection);
+        final var insertSpansAction = new InsertSpansAction(connection);
```

Comment:
Minor: Looking at this with the rename, I wonder if something like `InsertActivitySpansAction` rather than just `InsertSpansAction` (or renaming the function from `postActivities` to `postActivitySpans`) would help with clarity, since Aerie does have the concept of non-Activity Spans